### PR TITLE
Hw3

### PR DESCRIPTION
**Pull request by Nort07**

### CODE REVIEW

File: hm_task2.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Task 2
+
+toshiba = ["Steve Martin", " Whoopi Goldberg", " Dick Van Dyke", "Chevy Chase"]
+eleks = ["Robin Williams", "Eddie Murphy", "Adam Sandler"]
+toshiba.extend(eleks)
+print(f"\nEleks was taken over by Toshiba. All list of Toshiba members is:\n {toshiba} \n ")
+print(f"List of the old Eleks members is:\n {eleks}\n")
```
Comment (MarinaZelenska) [inline hm_task2.py:7]: Add  new additional line

File: hw_task4.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#  Task 4
+
+omnivores_group = ["Adam Sandler", "Steve Martin", "Whoopi Goldberg", "Dick Van Dyke", ]
+vegetarians_group = ["Eddie Murphy", "Whoopi Goldberg", "Dick Van Dyke", "Chevy Chase", ]
+omnivores_group.extend(vegetarians_group)
+result = omnivores_group
+print(f"List of guests who can eat and vegetables and herbs:\n{result}")
```
Comment (MarinaZelenska) [inline hw_task4.py:7]: I am not sure, but I think it is not a decision.
omnivores_guests = {'John', 'Lui', 'Andy', 'Sandy', 'Cler', 'Tanya', 'Fitch', 'Artur', 'Susan', 'Killian'}
vegetarians_guests = {'John', 'Lui', 'Andy', 'Sandy', 'Cler', 'Tanya', 'Fitch', 'Artur', 'Susan', 'Killian', 'Yulia', 'Steve', 'Gleb', 'Alex'}
only_vegetarians = vegetarians_guests.difference(omnivores_guests)
print(f'This people eat only vegetarians food: {only_vegetarians}')

File: hm_task3.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Task 3
+
+casino_blacklist = ["Adam Sandler", "Steve Martin", " Whoopi Goldberg", " Dick Van Dyke", ]
+poker_blacklist = ["Robin Williams", "Eddie Murphy", "Adam Sandler", "Bob Newhart", ]
+alcohol_blacklist = ["Chevy Chase", "Adam Sandler", "Ellen DeGeneres", "Bill Cosby", ]
+result = list(set(casino_blacklist) & set(poker_blacklist) & set(alcohol_blacklist))
+print(f"The person who is ni all black lists:{result} ")
```
Comment (MarinaZelenska) [inline hm_task3.py:7]: Try use sets and method intersection

**Comment by BohdanDovbysh1994** [comment]
General comment: all tasks should be in folder HW3

**Pull request closed without merging**